.github/workflows/ci.yml

```diff
@@ -40,3 +40,30 @@ jobs:
       - run: script/clone_all_rspec_repos
       - run: bundle install --binstubs --standalone
       - run: script/run_build
+
+  windows:
+    name: Ruby ${{ matrix.ruby }} (Windows)
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        ruby:
+          - 2.7
+          - 2.6
+          - 2.5
+          - 2.4
+          - 2.3
+          - 2.2
+          - 2.1.9
+      fail-fast: false
+    continue-on-error: ${{ matrix.ruby == '2.7' }}
```

Comment:
Why?

Reply:
Its out of date, but I need support approved first.

Reply:
I tried to do a similar thing on rspec-rails but I had to move it in step in order to continue-on-error properly https://github.com/rspec/rspec-rails/blob/e1a2ae7beffffa731a293f635320fba786af9189/.github/workflows/ci.yml#L101